### ISSUE TITLE
Duplicate MCM menus after merging

### ISSUE THREAD

**Issue by Hexwall** [matortheeternal/zedit#201]
Every time I add one more mod to the merged plugin and load my game some of my mods' mcm menus get duplicated. Is this a zMerge issue? Playing in SSE

**Comment by jfmherokiller** [comment]
yes this can happen when merging plugins in an already started game
